app/assets/javascripts/app/mix_panel.js

```diff
@@ -96,6 +80,19 @@ App.addChild('MixPanel', {
     });
   },
 
+  trackContributions: function(){
+    var from = '',
+        self = this;
+    if(window.location.href.indexOf('reward_id') > -1){
```

Comment:
Ths if can be inside the ```trackOnPage``` callback, since it's only needed when we have the page contributions/new.